bin/test/add-support-statement.test.js

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2023 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+const tap = require('tap')
+const proxyquire = require('proxyquire').noPreserveCache().noCallThru()
+const sinon = require('sinon')
+
+tap.test('Add Support Statement Scripting', (t) => {
```

Comment:
how is this getting run i see it in package.json but not in ci so effectively this will get out of date if not run regularly

Reply:
I put this in here to get a temperature check on if we wanted to have unit tests for our CI scripts. If we're cool with that, then I'll happily add running them to CI (so meta)